packages/itinerary-body/src/place-row.js

```diff
@@ -137,7 +137,7 @@ const PlaceRow = ({
             onClick={() => frameLeg({ isDestination, leg, legIndex, place })}
             title={messages.mapIconTitle}
           >
-            <Styled.MapIcon />
+            <Styled.MapIcon title="Show on map" />
```

Comment:
Are the rules wanting to have both an accessible title for the button AND and accessible title for the map icon? Or was the intent of the rule to make us provide an `alt` content for the map icon rather?

Reply:
I just double checked (changed `title` to `alt` and re-ran the tests) and indeed the rules require an inner title rather than an alt.

Reply:
Is there a documentation link for that rule?

Reply:
I believe it was this one https://dequeuniversity.com/rules/axe/4.3/region?application=axe-puppeteer